mlserver/server.py

```diff
@@ -56,6 +60,12 @@ def __init__(self, settings: Settings):
         self._configure_logger()
         self._create_servers()
 
+    def _create_response_cache(self) -> Optional[ResponseCache]:
+        if self._settings.cache_enabled:
+            return LocalCache(size=self._settings.cache_size)
+        else:
+            return None
```

Comment:
nit: I personally find `if / return` blocks more readable than `if / else`, e.g. :

```python
def _create_response_cache(self) -> Optional[ResponseCache]:
	if self._settings.cache_enabled:
	    return LocalCache(size=self._settings.cache_size)
	
	return None
```

But this is a completely personal opinion, so feel free to ignore it.

Reply:
Updated in [43dfc1c](https://github.com/SeldonIO/MLServer/pull/1440/commits/43dfc1c7587e738d377d437761fcb2bcade94854)